src/components/navigation-header/navigation-header.module.css

```diff
@@ -1,3 +1,6 @@
+@custom-media --nav-header-is-mobile (max-width: 1200px);
+@custom-media --nav-header-is-desktop (min-width: 1201px);
```

Comment:
This is where the bulk of the relatively minor refactor for more breakpoint flexibility happens.

The focal point here is create new `@custom-media` and using that where we previously might've used our more set-in-stone `mobile`, `tablet`, and `desktop` breakpoints.